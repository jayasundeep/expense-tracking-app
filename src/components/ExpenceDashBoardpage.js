import React from 'react';
import ExpenseList from './ExpensesList';

const ExpenceDashBoardpage = () => (
    <div>
        This is from ExpenceDashBoardpage!
        <ExpenseList />
    </div>
);

export default ExpenceDashBoardpage;