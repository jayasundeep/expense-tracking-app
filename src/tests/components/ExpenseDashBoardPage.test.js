import React from 'react';
import { shallow } from 'enzyme';
import ExpenceDashBoardpage from '../../components/ExpenceDashBoardpage';

test('should render Header component', () => {
    const wrapper = shallow(<ExpenceDashBoardpage />);
    expect(wrapper).toMatchSnapshot();
});