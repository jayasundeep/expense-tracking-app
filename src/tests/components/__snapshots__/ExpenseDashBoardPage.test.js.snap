// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
<div>
  This is from ExpenceDashBoardpage!
  <Connect(ExpenseList) />
</div>
`;
